readme.md

```diff
@@ -18,41 +18,67 @@ Enable Accelerated Mobile Pages (AMP) on your WordPress site.
+There's also a validator to help you ensure that AMP URLs look and function the same as in non-AMP.
 
-Your entire site can render as "Native AMP" if your theme calls `add_theme_support( 'amp' )`. There will only be one version of each URL: the AMP version. There won't be separate URLs with `/amp` or `?amp` appended. See this [wiki page](https://github.com/Automattic/amp-wp/wiki/Adding-Theme-Support#native-amp) for details and restrictions.
+This plugin supports all native Gutenberg blocks, and adds new blocks for AMP components. Including [amp-jwplayer](https://ampbyexample.com/components/amp-jwplayer/) and [amp-timeago](https://ampbyexample.com/components/amp-timeago/).
 
-Your theme can also use [Paired Mode](https://github.com/Automattic/amp-wp/wiki/Adding-Theme-Support#paired-mode), with your own custom templates for the AMP URLs.
+Use your own theme by selecting "Native" or "Paired" mode on the "AMP Settings" page. These have full support of commenting and widgets.
 
-"Native AMP" and "Paired Mode" add full support for commenting and widgets.
+"Native" mode will render your entire site as AMP. There will only be one version of each URL: the AMP version. There won't be separate URLs with `/amp` or `?amp` appended.
 
-If your theme doesn't support `'amp'`, this will use basic legacy post templates for AMP consumers like Google Search and Twitter. And when visiting the site, the AMP content won't normally appear without appending strings to the URL like `/amp` or `?amp`.
+"Paired" mode has an AMP URL and a non-AMP URL for the templates where AMP is enabled. For example, the non-AMP URL of `http://example.com/foo/` will have an AMP URL of `http://example.com/foo/?amp`.
+
+In both of these modes, you can select which templates should have AMP enabled. Choose a single template type like "Categories," a few templates, or your entire site.
+
+There's also a way for themes to set these modes by adding theme support of `'amp'`. Please see this document for more details.
+
+The themes Twenty Fifteen, Twenty Sixteen, and Twenty Seventeen support AMP. Simply select "Native" or "Paired" mode, and you can run your entire site as AMP without changing anything.
+
+"Classic" mode uses basic legacy post templates for AMP consumers like Google Search and Twitter. When visiting the site, the AMP content won't normally appear without appending strings to the URL like `/amp` or `?amp`.
+
+Posts, pages, media, and custom post types can have AMP-compatible versions in "Classic" mode. They're accessible by appending `/amp/` to your URL. For example, if your post URL is `http://example.com/2019/01/01/amp-on/`, you can access the AMP version at `http://example.com/2019/01/01/amp-on/amp/`. If you do not have [pretty permalinks](https://codex.wordpress.org/Using_Permalinks#mod_rewrite:_.22Pretty_Permalinks.22) enabled, you can do the same thing by appending `?amp=1`, i.e. `http://example.com/?p=123&amp=1`
```

Comment:
Good to mention that when in doubt, check the page source of the non-AMP version to find the `amphtml` link to identify what the AMP URL is for a given post. Also, this would be a good place to mention the AMP Validator Chrome extension which surfaces the AMP version of any URL while also showing if there are any validation errors: https://chrome.google.com/webstore/detail/amp-validator/nmoffdblmcmgeicmolmhobpoocbbmknc?hl=en